pkgs/build-support/docker/default.nix

```diff
@@ -738,6 +739,9 @@ rec {
     created ? "1970-01-01T00:00:01Z",
     # Optional bash script to run on the files prior to fixturizing the layer.
     extraCommands ? "",
+    # Optional bash script to run inside fakeroot environment.
+    # Could be used for changing ownership of files in customisation layer.
```

Comment:
Please (also) document this in the docs: https://github.com/NixOS/nixpkgs/blob/master/doc/builders/images/dockertools.section.md

Same for relevant info from the PR description and discourse message.

Reply:
Added documentation to that file, though not sure if it's best explanation as I'm not a native speaker of English. Please feel free to suggest changes or correct grammatical mistakes.